docs/002-basics-of-ci-with-actions.md

```diff
@@ -116,40 +116,40 @@ describe("<Home />", (): void => {
 
 ```
 
-The result of that last push to main should look like this image:
+The result of your last push to the main branch should resemble the following:
 
-![Actions overview showing a succesful workflow run](./images/success-nodejs-workflow.png)
+![Actions overview showing a successful workflow run](./images/success-nodejs-workflow.png)
 
-## 3 - Add Code Coverage to your workflow
+## 3 - Add code coverage to your workflow
 
-It is common when working on the CI part of your project to add more informations to the user, for example the tests "*code coverage*".
+When setting up CI for your project, it's common to provide additional information to users, such as test "*code coverage*".
```

Comment:
```suggestion
When setting up CI for your project, it's common to provide additional information to users, such as the test's "*code coverage*".
```

Reply:
I believe the original sentence was correct here. "test's code coverage" implies the code coverage is associated with a single test vs. a set of tests.

Reply:
Possible alternative:

```suggestion
When setting up CI for your project, it's common to provide additional information to users, such as code coverage stats for the project's tests.
```